pom.xml

```
@@ -24,7 +24,7 @@
         <kotlin.version>1.8.10</kotlin.version>
         <felles.version>1.20221202080911_bcf8f33</felles.version>
         <common-java-modules.version>2.2021.09.17_07.09-67428a6422cc</common-java-modules.version>
-        <kontrakter.version>2.0_20230307154139_81a3d26</kontrakter.version>
+        <kontrakter.version>3.0_20230314075921_27fa962-JAKARTA</kontrakter.version>
```

Nyeste versjon produsert fra master i familie-kontrakter:
```suggestion
        <kontrakter.version>2.0_20230313155154_f9438bd</kontrakter.version>
```